spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/ribbon/RibbonLoadBalancedRetryPolicyFactoryTests.java

```diff
@@ -36,6 +36,8 @@
 import org.springframework.cloud.netflix.ribbon.RibbonLoadBalancerClient.RibbonServer;
 
 import java.io.IOException;
+import java.net.SocketException;
```

Comment:
Accidental import?

Reply:
looks like a unit test was lost during the merge I will re-add that test

Reply:
If you add this test back soon, I can merge this.